### ISSUE TITLE
After update 2.6.5, the wordpress editor is open by default.

### ISSUE THREAD

**Issue by vaka2vaka** [elementor/elementor#8628]
![0101010](https://user-images.githubusercontent.com/48130019/61587969-705e8000-ab9c-11e9-9664-ad2900cdac20.jpg)

## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

## Description

If I turn in the page made in elementor to the wordpress editor, then the visual wordpress editor opens immediately.

In the previous version, when editing a page, “Edit with Elementor”-button was showing as default, while Edit with WordPress editor was an optional choice.

As it is now, the default WordPress visual editor is showing, which creates a lot of problems om pages I have created with Elementor, if I for example only want to change some things with Yoast…
Changes to the wordpress visual editor are not saved in this case.
On the elementor support forum, this is not just my problem.
## Steps to reproduce

You need to open the page made in elementor, in the visual editor WordPress.

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>
```Version:	5.2.2													
WP Multisite:	No							
Max Upload Size:	50 MB							
Memory limit:	512M													
Language:	ru-RU													
Debug Mode:	Inactive
Name:	OceanWP Child							
Version:	1.6.7.1556692004							
Author:	Nick							
Child Theme:	Yes							
Parent Theme Name:	OceanWP							
Parent Theme Version:	1.6.8							
Parent Theme Author:	Nick
```
</summary>

</details>

**Comment by shilo-ey** [comment]
Thanks for reporting @vaka2vaka 

We will look into it and release a patch soon.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.6.6

Feel free to update

Thanks!

**Comment by KimberlySwartz618** [comment]
So glad this is fixed!
